lib/modules/manager/helmfile/extract.ts

```diff
@@ -48,76 +56,82 @@ export function extractPackageFile(
     }
     logger.debug({ registryAliases }, 'repositories discovered.');
 
-    deps = doc.releases.map((dep) => {
-      let depName = dep.chart;
-      let repoName: string | null = null;
+    deps = await Promise.all(
+      doc.releases.map(async (dep) => {
+        let depName = dep.chart;
+        let repoName: string | null = null;
 
-      if (!is.string(dep.chart)) {
-        return {
-          depName: dep.name,
-          skipReason: 'invalid-name',
-        };
-      }
+        if (!is.string(dep.chart)) {
+          return {
+            depName: dep.name,
+            skipReason: 'invalid-name',
+          };
+        }
 
-      // If starts with ./ is for sure a local path
-      if (dep.chart.startsWith('./')) {
-        return {
-          depName: dep.name,
-          skipReason: 'local-chart',
-        };
-      }
+        // If it starts with ./ ../ or / then it's a local path
+        if (isLocalPath(dep.chart)) {
+          const needKustomize =
```

Comment:
we don't need to check here, as we skip that anyways. so revert

Reply:
Even though renovate will never want to update a local chart, I'm pretty sure we need to check here, since `helmfile deps` is run on the helmfile.yaml as a whole. When a helmfile.yaml contains a local chart using kustomizations and a non-local chart that renovate wants to update which does not use kustomization. We still need kustomize.  Please correct me if I'm not making any sense.

Reply:
I'm closing this conversation in favor of the new 🤗